assets/l10n/app_en.arb

```diff
@@ -59,6 +59,22 @@
   "@permissionsDeniedReadExternalStorage": {
     "description": "Message for dialog asking the user to grant permissions for external storage read access."
   },
+  "actionSheetOptionMuteTopic": "Mute topic",
```

Comment:
```
action_sheet: Support muting/unmuting/following topics
```

Bump on https://github.com/zulip/zulip-flutter/pull/1041#pullrequestreview-2453170881 :

> Thanks! Comments below, and I agree with your comment at [#348 (comment)](https://github.com/zulip/zulip-flutter/issues/348#issuecomment-2435903346) 🙂
> 
> > I think a design requirement of this feature is to also display mute/following states for each topic. At this point we can borrow that from the legacy mobile app.

The `Fixes:` line doesn't belong on this commit until we do that 🙂 or make it not part of #348.